package.json

```diff
@@ -45,7 +45,11 @@
     "react-dom": "^18.2.0",
     "react-paginate": "^8.2.0",
     "react-router-dom": "^6.10.0",
+<<<<<<< HEAD
+    "react-webcam": "^7.2.0",
+=======
     "react-router-hash-link": "^2.4.3",
+>>>>>>> e81082147c2a8c3c25c1ea905e80ecf248d5bdad
```

Comment:
These things in package.json appears to be remnants of the merge conflict resolution. The `<<<<<< HEAD`, `=======`, and `>>>>>>> e8108...` can all be removed.

I recommend removing your local `node_modules` and running a fresh `npm install` while keeping your `package.json` and `package-lock.json` intact.

Reply:
Agree